### PR TITLE
New package: g810-led-0.3.5

### PR DESCRIPTION
**Pull request by misuchiru03**
In response to this [request](https://github.com/void-linux/void-packages/issues/12431)

### CODE REVIEW

**Comment by misuchiru03** [comment]
This package will not build for musl.

**Comment by Johnnynator** [comment]
Can you please cleanup your branch so it only contains the g810-led commit.
Also instead of archs use
```shell
case $XBPS_TARGET_MACHINE_IN
    *-musl) broken="reason/build log";;
esac
```

File: srcpkgs/g810-led/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+homepage="https://github.com/MatMoul/g810-led/"
+distfiles="https://github.com/MatMoul/g810-led/archive/v${version}.tar.gz"
+checksum=6737d184fafcadcfd2e86d2cc6c92efb910f4abe97eebf6c52b0de9ab6864972
+
+g810-led-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}-${version}_${revision}"
+	pkg_install() {
+		vmkdir usr/lib
+		vcopy "lib/*.so.*" usr/lib
```
Comment (abenson) [inline srcpkgs/g810-led/template]: `-devel` should have '*.so', main package should have the versioned libs.

File: srcpkgs/g810-led/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	}
+}
+
+do_install() {
+	vbin bin/g810-led
+
```
Comment (abenson) [inline srcpkgs/g810-led/template]: remove the extra space

**Comment by misuchiru03** [comment]
Patched for musl build, cleaned-up commit, and removed `-devel` package.

File: srcpkgs/g810-led/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+build_style=gnu-makefile
+makedepends="hidapi-devel"
+short_desc="LED controller for Logitech G213,410,512,610,810,910,PRO keyboards"
+maintainer="Alex Childs <misuchiru03+void@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/MatMoul/g810-led/"
+distfiles="https://github.com/MatMoul/g810-led/archive/v${version}.tar.gz"
+checksum=6737d184fafcadcfd2e86d2cc6c92efb910f4abe97eebf6c52b0de9ab6864972
+
+#case $XBPS_TARGET_MACHINE in
```
Comment (Johnnynator) [inline srcpkgs/g810-led/template]: remove the three out commented lines

**Comment by misuchiru03** [comment]
Upstream merged PR to correct libc stdint variables--removed patch as no longer necessary.

**Comment by dragozet** [comment]
Thank you @misuchiru03 works great!

**Pull request merged**